Adapters/OpenQuestions.md

```diff
@@ -27,31 +24,13 @@ adapter.
 # ID Linking of the Objects Themselves
 When an adapter must alter (such as delete or update) a record in the system
 that it is connected to, there needs to be a method to link incoming requests
-from some  `other` system to the ids which exist in `this` system.  Consider the
-following possibilities.
-
-## Shared ID Pattern
-Objects share ids between systems.
-* Pros: Two way linking is possible
-* Cons: Ids need to exist in the same universe (i.e. GUID vs 64-bit integer).
-`this` system needs to allow the user to specify an ID upon creation.
-* If a native `upsert` operation exists in `this` system is present, use that.
-Otherwise, do a query for existence to determine if an insert or update
-operation is required.  `isNew` as metadata could be useful.
-
-## Foreign Key Pattern
-`this` system stores the id of the object in `other` system.
-* Assumes that an additional column/property can be created in `this` system.
-* Checks to see if an item exists where `otherId === incomingId`.  If yes,
-   update that item.  Otherwise create an item. `isNew` as metadata could be
-   useful.
-* Cons: Two way linking becomes complicated.  One or both systems will have to
- support the capability to add additional fields.
-
-## Third Party System Which Links IDs
-* Incoming object already is aware of the id in `this` system because
- this information was previously fetched from a master data system.
-  * See **OIH** for implementation.
+from some  `other` system to the ids which exist in `this` system.  
+
+The ID linking is solved by using **OIHApplicationDataRecords**.
```

Comment:
Is this the result of some other workgroup?

Reply:
Yes it is from the Data & Domain Model Workgroup. See: [Global Rules and Regulations for OMDMs](https://github.com/openintegrationhub/Data-and-Domain-Models/tree/master/MasterDataModels#4-global-rules-and-regulations-for-omdms)

Reply:
Can you add the link to the markdown file?

Reply:
Sure. But I will only work for those who are part of this project. All other visitors won't be able to see this markdown file.

Reply:
It's not clear to me why **Open** Integration Hub has documentation stored in private repos.